metrics/metrics_test.go

```diff
@@ -95,7 +97,8 @@ func TestPrometheusMetrics(t *testing.T) {
 
 			// Create the muxer for testing.
 			mx := http.NewServeMux()
-			pm := metrics.NewPrometheusMetrics(path, mx)
+			reg := prometheus.NewRegistry()
+			pm := metrics.NewPrometheusMetrics(path, "redis_operator", mx, reg)
```

Comment:
Pass the constant here `metricsNamespace`

Reply:
That constant is private in another package (main).

I've changed it to a different non-related namespace to show that it's configurable and that it changes the result of the metrics exposed.